tests/unittests/net/test_dhcp.py

```diff
@@ -815,16 +815,19 @@ def test_multiple_files(self):
 
 
 @pytest.mark.usefixtures("disable_netdev_info")
+@mock.patch("cloudinit.net.ephemeral._check_connectivity_to_imds")
```

Comment:
Are the changes in this file necessary? What was wrong with the `responses` code?